tests/transformers/models/test_causal_lm_models.py

```diff
@@ -16,6 +15,8 @@
 from QEfficient.utils.device_utils import get_available_device_id
 from tests.utils import get_cloud_ai_100_tokens, set_up
 
+TEST_CONFIG_FILE_PATH = os.path.join(ROOT_DIR, "tests", "config.json")
```

Comment:
Can you remove the `phi3` model card from the `config.json`, to test the CI pipeline? todo @quic-mamta for phi3 modeling debug.

Reply:
this is taken care in #63. Thanks